scene/main/viewport.cpp

```diff
@@ -643,7 +643,7 @@ void Viewport::_notification(int p_what) {
 					}
 				}
 
-				if (!motion_tested && camera && physics_last_mousepos != Vector2(1e20, 1e20)) {
+				if (!motion_tested && camera && physics_last_mousepos != Vector2(Math_INF, Math_INF)) {
```

Comment:
You can't use Math_INF as a literal value. This is not portable and may not even be possible to be read back as an infinite value later.

Reply:
`<cfloat>` defines `FLT_MAX`, `DBL_MAX` and so on for maximum finite floats, these should be portable.

Reply:
Good point, that would be better.

Reply:
I decided to go the boolean route.. I'll stay away from float comparison and magic numbers